includes/course-theme/class-sensei-course-theme-lesson.php

```diff
@@ -107,4 +111,35 @@ private function maybe_add_lesson_quiz_results_notice() {
 
 		$notices->add_notice( 'lesson_quiz_results', $text, __( 'Quiz completed', 'sensei-lms' ), $actions );
 	}
+
+	/**
+	 * Maybe add lesson prerequisite notice.
+	 */
+	private function maybe_add_prerequisite_notice() {
+		$user_id             = get_current_user_id();
+		$lesson_id           = \Sensei_Utils::get_current_lesson();
+		$lesson_prerequisite = \Sensei_Lesson::find_first_prerequisite_lesson( $lesson_id, $user_id );
+
+		if ( $lesson_prerequisite > 0 ) {
+			$user_lesson_status = \Sensei_Utils::user_lesson_status( $lesson_prerequisite, $user_id );
+
+			$prerequisite_lesson_link = '<a href="'
+				. esc_url( get_permalink( $lesson_prerequisite ) )
+				. '" title="'
+				// translators: Placeholder is the lesson prerequisite title.
+				. sprintf( esc_attr__( 'You must first complete: %1$s', 'sensei-lms' ), get_the_title( $lesson_prerequisite ) )
+				. '">'
+				. esc_html__( 'prerequisites', 'sensei-lms' )
+				. '</a>';
+
+			$text = ! empty( $user_lesson_status ) && 'ungraded' === $user_lesson_status->comment_approved
+				// translators: Placeholder is the link to the prerequisite lesson.
+				? sprintf( esc_html__( 'You will be able to view this lesson once the %1$s are completed. And a prerequisite is waiting for quiz grading.', 'sensei-lms' ), $prerequisite_lesson_link )
```

Comment:
How about `... once the prerequisites are completed and graded.`?

Reply:
I like that! Updated here: https://github.com/Automattic/sensei/pull/4509/commits/5565d7076e1af3c6f85f39de2b747a27b208f998